infrastructure/zk/src/server.ts

```diff
@@ -41,17 +39,22 @@ export async function apiNode(withDocker = false) {
 }
 
 export async function server(withDocker = false) {
-    // By the time this function is run the server is most likely not be running yet
-    // However, it does not matter, since the only thing the function does is depositing
-    // to the forced exit sender account, and server should be capable of recognizing
-    // priority operaitons that happened before it was booted
-    prepareForcedExitRequestAccount();
+    await prepareForcedExitAccountRequest();
     if (withDocker) {
         await docker.deployUp('server');
         return;
     }
 
-    await utils.spawn('cargo run --bin zksync_server --release');
+    await utils.spawn('cargo run --bin zksync_server --release&');
+}
+
+async function prepareForcedExitAccountRequest() {
+    if (process.env.ZKSYNC_ENV === 'dev') {
+        const address = '0x09a1eda29f664ac8f68106f6567276df0c65d859';
+        const privateKey = '0x082f57b8084286a079aeb9f2d0e17e565ced44a2cb9ce4844e6d4b9d89f3f595';
```

Comment:
Where this private key comes from? What do you think if in this case we use the JSON RPC Accounts (for dev only)? We could make the privateKey/address optional in the method signature and retrieve them directly in the `depositToForcedExitSenderAccount` method.
I don't really like hard-coding the private key (even if I know it was like that before)

Reply:
updated